erica/infrastructure/sqlalchemy/EricaAuftragSchema.py

```diff
@@ -15,7 +15,7 @@ class EricaAuftragSchema(AuditedSchemaMixin, BaseDbSchema):
     id = Column(UUID(as_uuid=True),
```

Comment:
We wanted to change the primary key from a UUID to a normal integer right?

Reply:
Also Part of the base repository pr because there are too many changes to that already

Reply:
Alrighty